infrastructure/kube/templates/keep-ecdsa/initcontainer/provision-keep-ecdsa/provision-keep-ecdsa.js

```diff
@@ -82,15 +82,14 @@ async function provisionKeepTecdsa() {
     console.log(`\n<<<<<<<<<<<< Authorizing Operator Contract ${bondedECDSAKeepFactory.address} >>>>>>>>>>>>`)
     await authorizeOperatorContract(operatorAddress, bondedECDSAKeepFactory.address, authorizer)
 
+    
     for (let i = 0; i < sanctionedApplications.length; i++) {
       const sanctionedApplicationAddress = sanctionedApplications[i]
 
-      console.log(`\n<<<<<<<<<<<< Create Sortition Pool for sanctioned application: ${sanctionedApplicationAddress} >>>>>>>>>>>>`)
-      const sortitionPoolContractAddress = await createSortitionPool(
-        sanctionedApplicationAddress
-      )
-
+      let sortitionPoolContractAddress = await getSortitionPool(sanctionedApplicationAddress)
+      
```

Comment:
Thoughts on adding a verification if a sortition pool exists?
```suggestion
      
  if (!sortitionPoolContractAddress || sortitionPoolContractAddress == ADDRESS_ZERO) {
    console.error(`missing sortition pool for application: [${applicationAddress}]`)
    continue
  }
```

Reply:
Yeah, we'll fail anyway but this will hopefully make things more obvious.